.pre-commit-config.yaml

```diff
@@ -88,3 +88,8 @@ repos:
       - id: ruff
         exclude: (thirdparty|cpp/sophus)/.*$
         args: [--fix, --exit-non-zero-on-fix]
+  - repo: https://github.com/myint/docformatter
```

Comment:
for some reason this link points to: https://github.com/PyCQA/docformatter
probably they moved the project

Reply:
Yeah they must've moved it